keycloak/keycloak-admin-ui
# Clients: Authorization -> Evaluate tab

## Motivation

https://marvelapp.com/prototype/5eg9ic7/screen/82302108

## Brief Description

Adds fields/selectable functionality to Authentication -> Evaluate tab

## Verification Steps


1. Go to Clients 
2. Create a new OIDC client and make sure "Client authentication" and "Authorization" switches are both enabled
3. Save the client and go to the Authorization -> Evaluate subtab
4. Verify that the fields are correct and work/populate as expected 


## Checklist:


## Additional Notes

Evaluate call and action group methods will be done in a separate PR
